test_make_prg.py

```diff
@@ -30,17 +30,17 @@ def test_answers():
     assert aseq.prg == "AAACGTGGTT"
 
     aseq = AlignedSeq("test/contains_n.fa")
-    assert aseq.prg == "AAACG 5 T 6 C 5 GGTT"
+    assert aseq.prg == "AAACG 5 C 6 T 5 GGTT"
 
     aseq = AlignedSeq("test/contains_RYKMSW.fa")
-    assert aseq.prg == "AAACG 5 T 6 C 5 GGTT"
+    assert aseq.prg == "AAACG 5 C 6 T 5 GGTT"
 
     aseq = AlignedSeq("test/contains_n_and_RYKMSW.fa")
-    assert aseq.prg == "AAACG 5 T 6 C 5 GGTT"
+    assert aseq.prg == "AAACG 5 C 6 T 5 GGTT"
 
     aseq = AlignedSeq("test/contains_n_and_RYKMSW_no_variants.fa")
     assert aseq.prg == "AAACGTGGTT"
 
-    with pytest.raises(Exception):
-        aseq = AlignedSeq("test/fails.fa")
+    #with pytest.raises(Exception):
```

Comment:
Why does this test no longer fail?

Reply:
Due to 0552d44  we now accept full colum of N's